src/Main.java

```diff
@@ -0,0 +1,25 @@
+public class Main {
+
+    public static void main(String[] args) {
+        //Create a method that will check if a given string is a palindrome.
+        // If it is it should print “This string is a palindrome”
+        // if not print “This string is not a palindrome”
+        String original = "racecar";
```

Comment:
You need to handle spaces, punctuations and Uppercase letters in original string before creating a reverse string. Your program should define strings "Was It A Cat I Saw" and "))Was It A Cat!!! I Saw?" as a palindromes